lib/zuora/objects/product.rb

```diff
@@ -3,11 +3,11 @@ class Product < Base
     has_many :product_rate_plans
 
     validates_inclusion_of :category, :in => ['DefaultCatalog'], :allow_nil => true
-    validates_length_of    :description, :maximum => 500, :allow_nil => true
+    validates_length_of :description, :maximum => 500, :allow_nil => true
 
-    validates_date_of      :effective_start_date, :effective_end_date
-    validates_length_of    :name, :maximum => 100, :allow_nil => true
-    validates_length_of    :sku, :maximum => 50, :allow_nil => true
+    validates_date_of :effective_start_date, :effective_end_date
```

Comment:
`product_rate_plan.rb` uses `validates_datetime_of` for `effective_start_date` and `effective_end_date`, so is `validates_date_of` correct here?